src/components/--Hero/Hero.stories.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import Hero from "./Hero";
+
+export default {
+  title: "Hero",
+  component: Hero,
+  args: {
+    imageUrl: "https://source.unsplash.com/random",
+  },
+  argTypes: {
```

Comment:
Found this in docs: since Storybook automatically creates controls(aka args) based on component props, I explicitly disable the unnecessary ones.